smartsim/settings/containers.py

```diff
@@ -53,7 +54,7 @@ class Singularity(Container):
         `system administrator <https://apptainer.org/docs/admin/1.0/configfiles.html#bind-mount-management>`_
 
 
-    :param image: local or remote path to container image, e.g. 'docker://sylabsio/lolcow'
+    :param image: local or remote path to container image, e.g. ``docker://sylabsio/lolcow``
```

Comment:
We need to hear more about this `lolcow` image.

Reply:
It's a fun example from their [documentation](https://apptainer.org/docs/user/main/quick_start.html#executing-commands):

```
$ apptainer exec docker://sylabsio/lolcow cowsay "Fresh from the internet!"
 _________________________
< Fresh from the internet! >
 -------------------------
        \   ^__^
         \  (oo)\_______
            (__)\       )\/\
                ||----w |
                ||     ||
```